deployment/pipeline-ec2-hadoop.md

```diff
@@ -8,14 +8,16 @@ In this section, we will do the following:
 #### Executing Jobs in EC2
 
 In the example below, we will demonstrate how to trigger and launch a job on an
-[EC2](http://aws.amazon.com/ec2/) instance. When EC2 is specified as the
-execution environment, the framework will launch instance(s) of the type
-specified in the region specified, before ultimately starting the job on the
+[EC2](http://aws.amazon.com/ec2/) instance. The job API allows for the specification
+of an execution environment other than the local host. When an EC2 cluster is
+specified as the execution environment, the framework will start the job on the
 new instances. While the job is running, the client machine can be shutdown and
 the job will continue. In the event that the client process terminates, you can
 reload the job and check its status.
 
-When defining your EC2 environment, you must specify a name, and a S3 path where
+An EC2 cluster is created in two steps: first, a configuration object is created,
+describing the cluster and how to access AWS. Second, the cluster is launched.
```

Comment:
describing the cluster => describe the properties for EC2 instances that are going to be used to form the cluster, like instance type, region where the instance is launched, security group name, etc.